pr2eus/robot-interface.l

```diff
@@ -1274,14 +1274,14 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
      (when base-controller-action-name
        (setq move-base-trajectory-action
              (instance ros::simple-action-client :init
-                       base-controller-action-name
+                       (if namespace (format nil "~A~A" namespace base-controller-action-name) base-controller-action-name)
```

Comment:
@Naoki-Hiraoka Are you sure about `"~A~A"` ? This assume user set `:base-controller-action-name "/base_controller/follow_joint_trajectory`, but there might be a change to set  `:base-controller-action-name "base_controller/follow_joint_trajectory`.

I think `/namespace//base_controller/follow_joint_trajectory` does not look nice, but it won't do any harm.
```
2.irteusgl$ (ros::subscribe "/hoge//hoge" std_msgs::String #'identity)
```
```
$ rostopic list
/hoge/hoge
```